code/modules/uplink/uplink_items.dm

```diff
@@ -35,18 +36,14 @@
 	var/desc = "item description"
 	/// Path to the item to spawn.
 	var/item = null
-	/// Alternative path for refunds, in case the item purchased isn't what is actually refunded (ie: holoparasites).
-	var/refund_path = null
 	/// Cost of the item.
 	var/cost = 0
-	/// Amount of TC to refund, in case there's a TC penalty for refunds.
-	var/refund_amount = 0
-	/// Whether this item is refundable or not.
-	var/refundable = FALSE
```

Comment:
Can we put this back and then instead of hardcoding the TC cost into two places, you add the element based on the presence of this boolean and can just feed the TC cost in that way?
There's 100% going to be bugs in the future where someone changes the TC cost in one place and not the other, otherwise.

Reply:
I don't know why I didn't think of that